src/Bicep.LangServer/Completions/CompletionPriority.cs

```diff
@@ -9,6 +9,8 @@ namespace Bicep.LanguageServer.Completions
     /// <remarks>The enum values are used to calculate completion sortText.</remarks>
     public enum CompletionPriority
     {
+        VeryHigh = 0,
```

Comment:
>VeryHigh [](http://example.com/codeflow?start=8&length=8)

@bhsupra and I discussed this over Teams. This seemed like the cleanest option to make the registry completions show up above everything else without regressing existing sorting.